scheds/rust/scx_layered/src/main.rs

```diff
@@ -123,6 +123,10 @@ lazy_static::lazy_static! {
 /// - NiceEquals: Matches if the task's nice value is exactly equal to
 ///   the pattern.
 ///
+/// - UserId: Matches if the task's effective user id matches the pattern.
```

Comment:
should this be `UserIdEquals`?

Reply:
Ah, I guess so. If it were me, I'd just do `UIDEquls` and `GIDEquals` but either way is fine.

Reply:
I think that's better, let me update.